stdlib/private/SwiftPrivateLibcExtras/Subprocess.swift

```diff
@@ -84,7 +85,7 @@ public func spawnChild(_ args: [String])
   let childStdin = posixPipe()
   let childStderr = posixPipe()
 
-#if os(Android) || os(Haiku)
+#if os(Android) || os(Haiku) || os(Fuchsia)
```

Comment:
The comment immediately below this could use some updating, since it's not just a codepath taken by Android anymore.